<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>580</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход из программы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Коннект к базе</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_shortname"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit_name"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Имя менеджера</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="2">
          <widget class="QPushButton" name="pushButton_table1_refresh">
           <property name="text">
            <string>Обновить таблицу</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Отчество менеджера</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" rowspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Фамилия менеджера</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLineEdit" name="lineEdit_family"/>
         </item>
         <item row="4" column="2" colspan="2">
          <widget class="QCheckBox" name="checkBox_admin">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>Менеджер имеет Администраторские права!</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" rowspan="2">
          <widget class="QCheckBox" name="checkBox_acticve">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Менеджер неактивен!</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_otchestvo"/>
         </item>
         <item row="2" column="1" rowspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Короткое имя менеджера</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="pushButton_insert_table1">
           <property name="text">
            <string>Записать в таблицу</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="pushButton_manager_delete">
           <property name="text">
            <string>Удалить данные о менеджере</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="5">
          <widget class="QTableWidget" name="tableWidget">
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Имя</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Отчество</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Фамилия</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Короткое имя</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuO_Ghjuhfvvt">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menuO_Ghjuhfvvt"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
